### PR TITLE
test!: replace mojaloop connector with sdk-scheme-adapter

### PR DESCRIPTION
**Pull request by kleyow**
The updating of names in the tests is because the `sdk-scheme-adapter` uses the updated fspiop regex for names i.e `^(?!\s*$)[\p{L}\p{gc=Mark}\p{digit}\p{gc=Connector_Punctuation}\p{Join_Control} .,''-]{1,128}$` while `mojaloop-connector` does not.

I'm not acquainted with pm4ml in production but will this regex cause problems down the line?

### CODE REVIEW

File: on-premise-deploy/testing-toolkit/environments/environment.json [hunk 1]
```diff
@@ -41,10 +42,10 @@
     "toIdType": "MSISDN",
     "toIdValue": "27713803912",
     "toDOB": "1970-01-01",
-    "toFirstName": "PayeeFirst",
```
Comment (mdebarros) [inline on-premise-deploy/testing-toolkit/environments/environment.json:44]: So this name will fail the tests?

**Review by mdebarros — CHANGES_REQUESTED** [review]
See my questions.

File: on-premise-deploy/testing-toolkit/environments/environment.json [hunk 2]
```diff
@@ -20,11 +20,12 @@
     "contentTypeQuotes": "application/vnd.interoperability.quotes+json;version=1.0",
     "currency": "USD",
     "fromDOB": "1984-01-01",
-    "fromFirstName": "Firstname-Test",
```
[the remark this answers is not in the document]
Comment (kleyow) [reply on-premise-deploy/testing-toolkit/environments/environment.json:23]: Correct.

File: on-premise-deploy/testing-toolkit/environments/environment.json (hunk 1 above)
Comment (kleyow) [reply on-premise-deploy/testing-toolkit/environments/environment.json:44]: Correct.

**Comment by kleyow** [comment]
Sam mentioned in todays meeting to just call this a breaking change and to just move forward with the fspiop standard. @mdebarros @elnyry-sam-k

**Comment by mdebarros** [comment]
> Sam mentioned in todays meeting to just call this a breaking change and to just move forward with the fspiop standard. @mdebarros @elnyry-sam-k

Ok, so no changes needed except making not of the breaking change?

**Pull request merged**